### PR TITLE
E2e test update

### PR DESCRIPTION
**Pull request by vara-bonthu**
*Issue #, if available:*

*Description of changes:*
- e2e test updates with new add-on approach

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/e2e/vpc/main.tf [hunk 1]
```diff
@@ -16,33 +16,24 @@
...
 }
 
 terraform {
   backend "s3" {
-    bucket = "terraform-ssp-github-actions-state"
-    key    = "e2e/vpc/terraform-main.tfstate"
-    region = "us-west-2"
+    bucket = ""
+    key    = ""
+    region = ""
```
Comment (askulkarni2) [inline test/e2e/vpc/main.tf:27]: Is this required?

Comment (vara-bonthu) [reply test/e2e/vpc/main.tf:27]: Yes, unfortunately we need those values to be empty otherwise it will throw an error

Comment (kcoleman731) [reply test/e2e/vpc/main.tf:27]: We sure about this? This doesn't seem right....can we just use variables here with defaults?

Comment (vara-bonthu) [reply test/e2e/vpc/main.tf]: @askulkarni2 Correction here. You are right that we may not need this anymore. I just updated the PR now.

**Review by Zvikan — APPROVED** [review]

**Pull request merged**